src/Eshopworld.Messaging/Eshopworld.Messaging.csproj

```diff
@@ -7,7 +7,8 @@
 
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>latest</LangVersion>
-
+    <DebugType>Full</DebugType>
```

Comment:
Might be something to look at in the future with "Portable" here so that we get the new PDB features (cross platform, vs code..etc etc)